core/src/scene/sceneLoader.cpp

```diff
@@ -364,7 +375,7 @@ void SceneLoader::loadStyles(YAML::Node styles, Scene& scene) {
         Node urlNode = styleNode["url"];
         if (urlNode) { logMsg("WARNING: loading style from URL not yet implemented\n"); } // TODO
 
-        scene.addStyle(std::unique_ptr<Style>(style));
+        scene.styles().push_back(std::unique_ptr<Style>(style));
```

Comment:
Should a check be happening to see if a style with the same name already exists?